sys/fs/tarfs/tarfs_vfsops.c

```diff
@@ -236,7 +236,7 @@ tarfs_checksum(struct ustar_header *hdrp)
 		checksum += *ptr;
 	for (idx = 0; idx < sizeof(hdrp->checksum); idx++)
 		checksum += 0x20;
-	for (ptr = (const unsigned char *)hdrp->typeflag;
```

Comment:
That's not got a & in front of it before? o.O

Reply:
Eh, ok, this is a char[1]. Sloppy coding practice upstream to be inconsistent like that here... this pair of loops doesn't use & for checksum and typeflag, whilst the pair of loops below does

Reply:
Yeah, it is odd that it is inconsistent upstream.